git-branchless-revset/src/builtins.rs

```diff
@@ -60,6 +61,21 @@ lazy_static! {
     };
 }
 
+#[instrument]
+fn fn_abandoned(ctx: &mut Context, name: &str, args: &[Expr]) -> EvalResult {
+    eval0(ctx, name, args)?;
+
+    let obsolete_commits = ctx.dag.query_obsolete_commits();
+    let abandoned_children = find_abandoned_children(
+        obsolete_commits,
+        // ?
```

Comment:
The `draft_commits` parameter is actually unused, so I opened https://github.com/arxanas/git-branchless/pull/744 to remove it.